Add `Bank.card_number`, `.card_expiry_date` and `.card_type`

AlexWayfer opened the pull request:
Like `Faker::Business`.

marocchino commented on lib/ffaker/bank.rb:
```diff
@@ -91,6 +91,18 @@ def iban(country_code: nil)
+      fetch_sample(CARD_NUMBERS)
+    end
+
+    def card_expiry_date
```
```suggestion
    def card_expiry_date(params={})
      FFaker::Time.date(params)
```
or maybe return only future date for convenient

AlexWayfer replied:
> or maybe return only future date for convenient

The card can be expired.

marocchino replied:
Ah, yes. You are right. Just passing params maybe?

AlexWayfer replied:
> Just passing params maybe?

Of course. I'm working on this.

AlexWayfer replied:
Maybe even a date from future as default.

AlexWayfer replied:
Done, with the 5 years range in the future as default.

marocchino commented on Changelog.md:
```diff
@@ -1,6 +1,6 @@
 ## development
 
-  - Add your change HERE
+  - Add `Bank.card_number`, `.card_expiry_date` and `.card_type` [@AlexWayfer]
```
leave original line for others. 😅 
```suggestion
  - Add your change HERE
  - Add `Bank.card_number`, `.card_expiry_date` and `.card_type` [@AlexWayfer]
```

AlexWayfer replied:
I see no sense, but as you wish. Done.

AlexWayfer commented:
I didn't see tests for params of `FFaker::Time.datetime`, so I've added them before tests for params for `FFaker::Bank.card_expiry_date`. Also, I've added (dynamically-generated) helper methods for operators like `<`, `>`, `<=`, `>=`. And for `between`. Also with `random` variation. And replaced for existing cases.
